opentelemetry-sdk/src/metrics/meter.rs

```diff
@@ -479,6 +513,75 @@ impl InstrumentProvider for Meter {
     }
 }
 
+/// Validation policy for instrument
+#[derive(Clone, Copy)]
+enum InstValidationPolicy {
```

Comment:
I'd prefer to see `Inst` spelled out to `Instrument`, but that's just me.  (e.g. is his "Instance" validation policy or "Instrument" validation policy?)

Reply:
thanks! there is an InstProvider, so I tried to be consistent. but rename to InstrumentValidationPolicy